packages/wonder-blocks-i18n/src/components/i18n-inline-markup.js

```diff
@@ -110,17 +110,27 @@ type Props = {|
 |};
 
 export class I18nInlineMarkup extends React.PureComponent<Props> {
+    /**
+     * If an error occurs, we either call the onError prop, or throw the
+     * error.
+     */
+    handleError(error: Error): React.Node {
+        const {onError} = this.props;
```

Comment:
**suggestion:** I can't comment on it because it was not changed in this PR, but could you add some comments to the `onError` prop to explain how it is used?